Sources/MapboxDirections/RouteOptions.swift

```diff
@@ -393,6 +403,14 @@ open class RouteOptions: DirectionsOptions {
     }
     private var _initialManeuverAvoidanceRadius: LocationDistance?
     
+    /**
+     :nodoc:
+     Toggle whether to return calculated toll cost for the route, if data is available.
+     
+     Default value is `false`.
```

Comment:
Mention that, if this property is set to `true`, each resulting route’s `Route.tollPrices` property is populated with toll prices.